src/styles.tsx

```diff
@@ -0,0 +1,43 @@
+export const color={
+    primary:"#00b4d8",
+    secondary:"#00b4d8",
+    danger:"#00b4d8",
+    success:"#00b4d8",
+    warning:"#00b4d8",
+};
+export const boxHW = {
+      height:{
+        small:"10rem",
+        medium:"30rem",
+        large:"50rem"
+      },
+      width:{
+        small:"10rem",
+        medium:"30rem",
+        large:"50rem"
+      }
+};
+export const headingHW = {
+      height:{
+        small:"3rem",
+        medium:"4rem",
+        large:"6rem"
+      },
+      width:{
+        small:"40rem",
+        medium:"50rem",
+        large:"70rem"
+      }
+};
+export const buttonHW = {
+      height:{
+        small:"2rem",
+        medium:"3rem",
+        large:"5rem"
+      },
+      width:{
+        small:"10rem",
+        medium:"15rem",
+        large:"20rem"
+      }
+};
```

Comment:
```suggestion
export const SPACE = {
  XXS: '0.2rem',
  XS: '0.4rem',
  S: '0.8rem',
  M: '1rem',
  L: ...,
  XL: ...,
  XXL...
}
```

Reply:
same space can be used in padding, margin etc

Reply:
yup, I will make space constant.